### PR TITLE
fixes tabular table, required fields label, related widget wrapper overflow

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

|BEFORE|AFTER|
|---|---|
|screen shot 2015-11-18 at 14 25 48|screen shot 2015-11-18 at 14 25 28|
|screen shot 2015-11-18 at 14 05 34|screen shot 2015-11-18 at 14 27 37|
|screen shot 2015-11-18 at 14 34 46|screen shot 2015-11-18 at 14 35 26|

### CODE REVIEW

**Comment by vxsx** [comment]
:+1:

File: djangocms_admin_style/sass/components/_forms.scss [hunk 1]
```diff
@@ -737,6 +737,7 @@ form select[multiple] {
 
 //related-widget
 .related-widget-wrapper {
+    overflow: visible;
```
Comment (FinalAngel) [inline djangocms_admin_style/sass/components/_forms.scss]: @lory87 I would recommend for the future to always add comments like:

``/* required in /cms/pages... to ensure outline is not cut off */``

File: djangocms_admin_style/sass/components/_forms.scss [hunk 2]
```diff
@@ -231,8 +234,8 @@ form {
         width: auto !important;
         margin-bottom: 5px;
         &.required {
-            color: $gray !important;
-            font-weight: normal !important;
+            color: $black !important;
```
Comment (FinalAngel) [inline djangocms_admin_style/sass/components/_forms.scss]: same here ``/* make sure we can differentiate required from normal side */``

**Comment by FinalAngel** [comment]
I'm okay with the merge, but please add more comments on future PR's. I had the problem that I changed styles from time to time but was unsure about relations. It's also okay to do this recursively on a separate PR and add the PR numbers with the view cases for example

**Pull request merged**